damnit/backend/extract_data.py

```diff
@@ -204,6 +204,10 @@ def extract_and_ingest(self, proposal, run, cluster=False,
 
             python_cmd = [sys.executable, '-m', 'damnit.backend.extract_data',
                           '--cluster-job', str(proposal), str(run), run_data.value]
+            if len(match) > 0:
```

Comment:
I don't think you need this condition - if `m` is empty, the loop won't do anything anyway. :slightly_smiling_face:

Reply:
Ah true, removed in abf6e13.